Configure

```diff
@@ -240,6 +240,7 @@ my @disablables = (
     "autoalginit",
     "autoerrinit",
     "bf",
+    "bp",
```

Comment:
I agree, but at the same time PAIRING_GROUP looks huge. What about pbc_, any confusion with the PBC library?